html/browsers/sandboxing/sandbox-navigation-timing-2.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Sandbox Navigation Timing</title>
+    <script src=/resources/testharness.js></script>
+    <script src=/resources/testharnessreport.js></script>
+  </head>
+  <body>
+    <script>
+    const resourcePath =
```

Comment:
I guess this works, but this might have been simpler:
```
const resourceURL = new URL("resources/", location);
```

and then later:
```
iframe.src = new URL("post-sandbox-state-to-parent.html", resourceURL);
```

or so.  Similar elsewhere.